docs/en/deploy/Running-Alluxio-On-a-HA-Cluster.md

```diff
@@ -0,0 +1,359 @@
+ master_hostname_3:19998`
+
+- When connecting to an Alluxio HA cluster using Zookeeper,
+the following properties are needed to connect to Zookeeper to get the leading master information.
+  - `alluxio.zookeeper.enabled=true`
+  - `alluxio.zookeeper.address=<ZOOKEEPER_ADDRESS>`
+    - The ZooKeeper address must be specified when `alluxio.zookeeper.enabled` is enabled.
+    - Multiple ZooKeeper addresses can be specified by delimiting with commas
+
+When the application is configured with these parameters, the Alluxio URI can be simplified to
+`alluxio:///path`, since the HA cluster information is already configured.
+
+For example, if using Hadoop, you can configure the properties in `core-site.xml`, and then use the
+Hadoop CLI with an Alluxio URI.
+
+```bash
+hadoop fs -ls alluxio:///directory
+```
+
+### Setting Alluxio Service with URL Authority {#ha-authority}
+
+Users can fully specify the HA cluster information in the URI to connect to an Alluxio HA cluster.
+Configuration derived from the HA authority takes precedence over all other forms of configuration, e.g. site properties or environment variables.
+
+- When using embedded journal, use `alluxio://master_hostname_1:19998,
+master_hostname_2:19998,master_hostname_3:19998`
+- When using Zookeeper leader election, use `alluxio://zk@<ZOOKEEPER_ADDRESS>`.
+
+For many applications (e.g., Hadoop, HBase, Hive and Flink), you can use a comma as the
+delimiter for multiple addresses in the URI, like `alluxio://master_hostname_1:19998,master_hostname_2:19998,master_hostname_3:19998`
+and `alluxio://zk@zkHost1:2181,zkHost2:2181,zkHost3:2181/path`.
+
+For some other applications (e.g., Spark), you need to use semicolons as the delimiter for multiple
+addresses, like `alluxio://master_hostname_1:19998;master_hostname_2:19998;master_hostname_3:19998`
+and `alluxio://zk@zkHost1:2181;zkHost2:2181;zkHost3:2181/path`.
+
+## Common Operations
+
+Below are common operations to perform on an Alluxio cluster.
+
+### Stop Alluxio
+
+To stop an Alluxio service, run:
+
+```bash
+./bin/alluxio-stop.sh all
+```
+
+This will stop all the processes on all nodes listed in `conf/workers` and `conf/masters`.
+
+You can stop just the masters and just the workers with the following commands:
+
+```bash
+./bin/alluxio-stop.sh masters # stops all masters in conf/masters
+./bin/alluxio-stop.sh workers # stops all workers in conf/workers
+```
+
+If you do not want to use `ssh` to login to all the nodes and stop all the processes, you can run
+commands on each node individually to stop each component. For any node, you can stop a master or worker with:
+
+```bash
+./bin/alluxio-stop.sh master # stops the local master
+./bin/alluxio-stop.sh worker # stops the local worker
+```
+
+### Restart Alluxio
+
+Starting Alluxio is similar. If `conf/workers` and `conf/masters` are both populated, you can start
+the cluster with:
+
+```bash
+./bin/alluxio-start.sh all SudoMount
```

Comment:
I don't think we should add `SudoMount` for restart?

Reply:
Good point, removed